Add image processor module.

monhayek opened the pull request:
Unit tests will be added after I remove some internal decencies.

zhouwubai commented:
@sasantavakkol @yaoyichi FYI, btw, seems I can not add reviewers

sasantavakkol commented on main.py:
```diff
@@ -0,0 +1,16 @@
+def hello_world(request):
```
What is this function for? If for cloud functions deployment, please choose a name reflecting this.

monhayek committed:
Create util

monhayek committed:
Delete util

monhayek committed:
Create image_processor.py

This module uses Google Vision API to detect the text in a raster map, and send the textual results to Google NLP API to identify addresses and locations. The resulting entities are then consumed by the Google Geocoding API to determine where the map belongs to.

The Vision API has a limit of 20 MB file size, and some of the maps reach up to 300 MB in size,  causing requests timeouts. This module is a starting point to optimize for this problem using a recursion. The main function  takes in the image and divide it into two parts each time, once horizontally and once vertically until the 20 MB size limit is reached.

SasanTV commented on util/image_processor.py:
```diff
@@ -0,0 +1,229 @@
+"""Image processing module for OCR.
```
Please add an Apache license header. You can find an example in geolocalizer.py.
Please also add more details to the comment describing this module. You can copy from your commit descriptions.

monhayek committed:
Update image_processor.py

monhayek committed:
Update comments/documentation.
